test/e2e/supportbundle_test.go

```diff
@@ -56,18 +57,19 @@ func testSupportBundle(name string, t *testing.T) {
 	}
 	defer teardownTest(t, data)
 
-	var podName, podPort, tokenPath string
+	var podName, tokenPath string
+	var podPort int
```

Comment:
prefer to declare variable in a single line, line 60 and 61 can be combined
var ( 
    d int 
    e string
  )